app/views/insured/families/home.html.erb

```diff
@@ -1,58 +1,91 @@
-<%= javascript_pack_tag "enrollment" %>
-<%= stylesheet_pack_tag "enrollment" %>
-
-<% content_for :navigation do %>
-  <%= render partial: 'ui-components/v1/navs/families_navigation' %>
-<% end %>
-
-<div class="col-xs-12">
-  <div id="plan-summary"></div>
-</div>
-
-<div id='account-detail'>
-  <% @person.active_employee_roles.each do |employee_role|%>
-    <%= render 'employer_panel', employee_role: employee_role if newhire_enrollment_eligible?(employee_role) %>
+<% if @bs4 %>
+  <% if pundit_allow(Family, :can_view_entire_family_enrollment_history?) %>
+    <% enrollments = @all_hbx_enrollments_for_admin %>
+  <% else %>
+    <% enrollments = @hbx_enrollments %>
   <% end %>
-
-  <%= render 'sep_message', locals: {family_id: @family.id.to_s} if @active_sep.present? %>
-
-  <div class="my-account-page">
-    <h1 class="heading-text mt-0"><%= l10n("my_site_short_name", site_short_name: site_short_name) %></h1>
-  </div>
-
-  <div class="row">
-    <div class="col-md-8">
-      <%= render partial: 'employers_selection' if @person.has_multiple_active_employers? %>
-      <%= render partial: 'qle_detail' %>
-      <%= render partial: 'qle_detail_for_existing_sep' if @active_sep.present? %>
-
-      <% if pundit_allow(Family, :can_view_entire_family_enrollment_history?) %>
-        <% enrollments = @all_hbx_enrollments_for_admin %>
-      <% else %>
-        <% enrollments = @hbx_enrollments %>
+  <div class="w-100">
+    <%= render partial: 'shared/family_side_nav' %>
+    <div class="pl-2">
+      <% @person.active_employee_roles.each do |employee_role|%>
+        <%= render 'employer_panel', employee_role: employee_role if newhire_enrollment_eligible?(employee_role) %>
       <% end %>
 
+      <%= render 'sep_message', locals: {family_id: @family.id.to_s} if @active_sep.present? %>
+      <h1><%= l10n("insured.my_coverme_header") %></h1>
+
+      <%= render 'right_column' %>
+      <h2><%= enrollments.last.coverage_year %> <%= l10n('coverage').titleize %></h2>
```

Comment:
This breaks the page if the family does not have any enrollments, you need to add a conditional around the whole enrollment section to check for the enrollments existance. if no plans exist, don't show anything there.

Reply:
wrapped inside an if: https://github.com/ideacrew/enroll/pull/4022/commits/8d2e0ecdcb50420a27185714173553a5e9dc36fd#diff-11d363b5759f355e48892811af2e745d24def1f9b957b65821cead88bf0fc716R18-R24